src/managers/selection-manager/selection-manager.js

```diff
@@ -0,0 +1,188 @@
+/**
```

Comment:
Updated the JSDocs. 

Double-checked the use of the aria-checked vs aria-selected. In the case of the checkbox, because it is role="checkbox" it needs to have the aria-checked attribute set, which also allows for the "mixed" state. For the child list-items which have the role="option" attribute set, they require the aria-selected attribute set. In our implementation, this is correct to default the masterAttribute parameter to "aria-checked", and the itemAttribute default to "aria-selected"